docs/content/connectors/oracle-cdc.md

```diff
@@ -288,10 +288,10 @@ CREATE TABLE products (
     schema_name STRING METADATA FROM 'schema_name' VIRTUAL, 
     table_name STRING METADATA  FROM 'table_name' VIRTUAL,
     operation_ts TIMESTAMP_LTZ(3) METADATA FROM 'op_ts' VIRTUAL,
-    id INT NOT NULL,
```

Comment:
The metadata column name accepts both lowercase and uppercase, could we add a note for physical column that `The original oralce column name is uppercase by default.`

Reply:
Thanks @leonardBang , I will do it.